components/form-builder/app/responses/DownloadTableDialog.tsx

```diff
@@ -102,6 +113,16 @@ export const DownloadTableDialog = ({
       return;
     }
 
+    // TEMP
+    if (tempType === "report" && descriptionRef.current === "") {
```

Comment:
What is `tempType` here?

Reply:
It acts to identify which dialog to show/hide related content. Named temp to make it very clear that the type and related is just temporary. The refactor will have two dialogs etc.